src/DependencyInjection/Compiler/AddResourcesPathsPass.php

```diff
@@ -41,7 +41,7 @@ private function getResourcesPath(ContainerBuilder $container)
         $rootDir = dirname($container->getParameter('kernel.root_dir'));
 
         foreach ($container->getParameter('kernel.bundles') as $name => $class) {
-            if ('Contao\CoreBundle\HttpKernel\Bundle\ContaoModuleBundle' === $class) {
+            if (is_a($class, 'Contao\CoreBundle\HttpKernel\Bundle\ContaoModuleBundle', true)) {
```

Comment:
I have read it and I still don't understand @Toflar's comment. The first argument has to be an object, hasn't it?